ui/admin/app/components/form/auth-method/oidc/index.hbs

```diff
@@ -155,22 +155,27 @@
           {{#each @model.attributes.signing_algorithms as |algorithm i|}}
             <body.row as |row|>
               <row.cell>
-                <form.select
+
+                <Hds::Form::Select::Field
                   @value={{algorithm.value}}
-                  @error={{@model.errors.attributes_signing_algorithms}}
-                  @contextual={{true}}
-                  @onChange={{fn (mut algorithm.value)}}
-                  as |field|
+                  @isInvalid={{@model.errors.attributes_signing_algorithms}}
+                  disabled={{form.disabled}}
+                  {{on 'change' (set-from-event algorithm 'value')}}
+                  as |F|
                 >
-                  <field.label>{{t 'form.algorithm.label'}}</field.label>
-                  <field.field as |select|>
-                    {{#each this.signingAlgorithms as |algorithm|}}
-                      <select.option
-                        @value={{algorithm}}
-                      >{{algorithm}}</select.option>
+                  <F.Label class='sr-only'>{{t
```

Comment:
Is this class necessary here?

Reply:
Yes.  This hides the label visually.  It's necessary for accessibility, but visually redundant because of the fields.